prisma/schema.sql

```diff
@@ -83,11 +86,35 @@ CREATE TABLE adjudications (
   cumulative_mark INTEGER NOT NULL,
   audio_url VARCHAR(255),
   notes TEXT,
-  special_award VARCHAR(255),
   performance_id INTEGER NOT NULL,
+  user_id INTEGER NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  FOREIGN KEY(performance_id) REFERENCES performances(id),
+  FOREIGN KEY(user_id) REFERENCES users(id)
+);
+
+-- Create awards table
+CREATE TABLE awards (
+  id SERIAL PRIMARY KEY NOT NULL,
+  title VARCHAR(255) NOT NULL,
+  is_finalized BOOLEAN NOT NULL DEFAULT FALSE,
+  user_id INTEGER NOT NULL,
   created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  FOREIGN KEY(performance_id) REFERENCES performances(id)
+  FOREIGN KEY(user_id) REFERENCES users(id)
+);
+
+-- Create awards_performances table
+CREATE TABLE awards_performances (
+  id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
Prisma's doc had this id in addition to the ids of the 2 associated tables in their documentation but it seems a bit redundant

Reply:
think it's fine